pkg/rbac/userData.go

```diff
@@ -408,9 +409,30 @@ func (shared *SharedData) isClusterScoped(kindPlural, apigroup string) bool {
 	// lock to prevent checking more than one at a time and check if cluster scoped resources already in cache
 	shared.csrCache.lock.Lock()
 	defer shared.csrCache.lock.Unlock()
-	_, ok := shared.csResourcesMap[Resource{Apigroup: apigroup, Kind: kindPlural}]
+	var ok bool
+	resource := Resource{Apigroup: apigroup, Kind: kindPlural}
+	_, ok = shared.csResourcesMap[resource]
 	if ok {
 		klog.V(9).Info("resource is ClusterScoped ", kindPlural, " ", apigroup, ": ", ok)
+	} else {
+		// check if it is a cluster-scoped resource in this list
+		openshiftClusterScopedRes := map[Resource]struct{}{
+			{Apigroup: "authorization.openshift.io", Kind: "clusterroles"}:      {},
+			{Apigroup: "", Kind: "clusterroles"}:                                {},
+			{Apigroup: "console.openshift.io", Kind: "consoleexternalloglinks"}: {},
+			{Apigroup: "console.openshift.io", Kind: "consolelinks"}:            {},
+			{Apigroup: "console.openshift.io", Kind: "consolenotifications"}:    {},
+			{Apigroup: "console.openshift.io", Kind: "consoleyamlsamples"}:      {},
+			{Apigroup: "project.openshift.io", Kind: "projects"}:                {},
+			{Apigroup: "", Kind: "projects"}:                                    {},
+			{Apigroup: "project.openshift.io", Kind: "projectrequests"}:         {},
+			{Apigroup: "", Kind: "projectrequests"}:                             {},
+			{Apigroup: "oauth.openshift.io", Kind: "useroauthaccesstokens"}:     {},
+		}
+		if _, ok = openshiftClusterScopedRes[resource]; ok {
+			klog.V(9).Info("resource is openshiftClusterScoped ", kindPlural, " ", apigroup, ": ", ok)
```

Comment:
Apigroup isn't logged correctly.
```suggestion
			klog.V(9).Infof("resource is openshiftClusterScoped %+v", resource)
```